javascript/sdk/lib/controllers/iam.ts

```diff
@@ -48,22 +48,30 @@ export default class IAMController {
   }
 
   /** Update properties on an account */
-  async updateAccount(mode: Mode, options: RequestOptions = {}) {
+  async updateAccount(
```

Comment:
like the other update functions, I'm gonna return the account in the response now